test-it/karma/javascriptinvoke/Async.Test.js

```diff
@@ -335,6 +335,50 @@ describe('A JavaScript function invoke', function () {
         });
     });
 
+    describe('errors retrieving completion callback twice without a try/catch are printed to console when', function () {
+        beforeEach(function () {
+            spyOn(console, 'error');
+            test = async function () {
+                var viName = 'SingleFunction';
+                var runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, jsAsyncFunctionsUrl);
+                var viPathParser = vireoRunner.createVIPathParser(vireo, viName);
+                var viPathWriter = vireoRunner.createVIPathWriter(vireo, viName);
+                viPathWriter('input', 7);
+                vireoRunner.enqueueVI(vireo, viName);
+
+                const {rawPrint, rawPrintError} = await runSlicesAsync();
+                expect(rawPrint).toBeEmptyString();
+                expect(rawPrintError).toBeEmptyString();
+                expect(console.error).toHaveBeenCalled();
+                expect(viPathParser('error.status')).toBeTrue();
+                expect(viPathParser('error.code')).toBe(44300);
+                expect(viPathParser('error.source')).toMatch(/retrieved more than once/);
+                expect(viPathParser('return')).toBe(0);
+            };
+        });
+        it('using the completion callback', async function () {
+            window.SingleFunction = function (input, jsapi) {
+                var completionCallback = jsapi.getCompletionCallback();
+                jsapi.getCompletionCallback();
+                // TODO mraj this allows completing after an exception is thrown, this should not be allowed.
```

Comment:
Can you remove these TODOs from here and above? The strategy has changed such that if the getCompletionCallback throws an error and a user catches it, it does make sense for the user to complete the test (we should remove the comment in the above tests).